clients/java/order-handling/pom.xml

```
@@ -13,7 +13,7 @@
   <properties>
     <camunda.platform.version>7.20.0</camunda.platform.version>
     <slf4j.version>1.7.36</slf4j.version>
-    <jaxb.version>2.3.6</jaxb.version>
+    <jaxb.version>4.0.4</jaxb.version>
```

🔧 This version needs to be also updated in case you consider the [patch update](https://github.com/camunda/camunda-bpm-platform/pull/4069#discussion_r1523458104) in `camunda-bpm-platform`
```suggestion
    <jaxb.version>4.0.5</jaxb.version>
```